src/screens/OrgList/OrgList.tsx

```diff
@@ -268,6 +286,34 @@ function orgList(): JSX.Element {
   };
 
   const debouncedHandleSearchByName = debounce(handleSearchByName);
+  const handleSorting = (option: string): void => {
+    setSortingState({
+      option,
+      selectedOption: t(option),
+    });
+  };
+
+  const sortOrgs = (
```

Comment:
Is there any query available in the talawa-api which sorts this in backend itself? If yes, please the GraphQL query instead of sorting it on the frontend.

Reply:
Check `sort` in the query resolvers for the query used to fetch the organizations data.

Reply:
Please check the query arguments for `ORGANIZATION_CONNECTION_LIST` query.

Reply:
@beingnoble03, I actually figured that out earlier myself. I modified all the parameters as needed, but I'm not receiving any response from the API. If you're free, I'd like to discuss this issue further on Slack.
Thank you